quickwit/quickwit-query/src/elastic_query_dsl/match_query.rs

```diff
@@ -46,6 +48,11 @@ pub(crate) struct MatchQueryParams {
     pub(crate) operator: BooleanOperand,
     #[serde(default)]
     pub(crate) zero_terms_query: MatchAllOrNone,
+    /// Currently ignored for Match queries (as we do not handle fuzziness) but used in
+    /// MatchBoolPrefixQuery. In the latter, it will control the maximum number of terms to
+    /// match having prefix the last term of the query.
```

Comment:
Not sure if this is clearer but I tried.

```suggestion
    /// Currently ignored for Match queries (as we do not handle fuzziness) but used in
    /// MatchBoolPrefixQuery. In the latter, it limits the maximum number of terms starting with the prefix
    /// defined by the last term of the query.
```